src/modules/file/file.usecase.ts

```diff
@@ -36,6 +36,7 @@ import { WorkspaceAttributes } from '../workspaces/attributes/workspace.attribut
 import { Folder } from '../folder/folder.domain';
 import { getPathFileData } from '../../lib/path';
 import { isStringEmpty } from '../../lib/validators';
+import { Op } from 'sequelize';
```

Comment:
Could you try to use this in the repository? You can find the reason in this comment: https://github.com/internxt/drive-server-wip/pull/428#discussion_r1837767776